.github/workflows/codeql.yml

```diff
@@ -0,0 +1,64 @@
+name: CodeQL
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+    - master
+  schedule:
+  - cron: '19 2 * * 4'
```

Comment:
push and pull_request triggers should be enough, I don't think it needs a schedule trigger